cartridges/int_klaviyo_sfra/cartridge/scripts/klaviyo/viewedProductHelpers.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/* Script Modules */
+var fullProductModel = require('*/cartridge/models/product/fullProduct');
+var productHelper = require('*/cartridge/scripts/helpers/productHelpers');
+
+
+function getProductPrices(product) {
```

Comment:
Same comment as for the `getProductPrices` function for sitegen - requesting a short docstring here

Reply:
can do

Reply:
Done.